coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -0,0 +1,35 @@
+from coalib.results.result_actions.ShowPatchAction import ShowPatchAction
+from coalib.results.result_actions.ResultAction import ResultAction
+import os.path
+
+
+class IgnoreResultAction(ResultAction):
+
+    SUCCESS_MESSAGE = "Result ignored successfully."
+
+    is_applicable = staticmethod(ShowPatchAction.is_applicable)
```

Comment:
So ignoring a result is actually ignoring a patch? Some results just have a open editor action right? Those cannot be ignored?